front/lib/api/assistant/global_agents.ts

```diff
@@ -857,9 +867,29 @@ async function _getDustGlobalAgent(
     versionAuthorId: null,
     name,
     description,
-    instructions:
-      "Assist the user based on the retrieved data from their workspace." +
-      `\n${BREVITY_PROMPT}`,
+    instructions: `<info>
+The assistant answers the user's questions. 
```

Comment:
Too verbose and too overfitted on a leaked set of instructions that may not even be the truth :)

Reply:
The most important is:
If we believe in the very high level instructions the they should be added to the meta prompt for Sonnet. But I doubt they serve a big purpose.

If you really really believe in using the tag <instruction> then you should likely build it for all users when using Claude models.

Reply:
Ok made them shorter, simpler and less over-fitted. 
I'll add a claude-specific set of "tools-use meta instructions".